src/goalServices/reduceGoals.ts

```diff
@@ -425,6 +474,13 @@ export function reduceGoals(
             currentValue.dataValues.prompts || [],
             (existingGoal.prompts || []) as IPrompt[],
           );
+          // This will be for review on the report.
+          existingGoal.promptsForReview = reducePromptsForReview(
+            currentValue.dataValues.grant.recipientId,
+            currentValue.dataValues.grant.recipientNameWithPrograms,
+            currentValue.dataValues.prompts || [],
+            existingGoal.promptsForReview,
```

Comment:
with the inconsistent check/use of .dataValues, its hard to know if all code paths will run as expected. Does there need to be checks here to not use .dataValues?